Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReport.c

```diff
@@ -440,18 +446,37 @@ uintptr_t *bsg_kscrw_i_getBacktrace(
     }
 
     int actualSkippedEntries = 0;
-    int actualLength = bsg_ksbt_backtraceLength(machineContext);
-    if (actualLength >= BSG_kStackOverflowThreshold) {
-        actualSkippedEntries = actualLength - *backtraceLength;
-    }
 
-    *backtraceLength =
-        bsg_ksbt_backtraceThreadState(machineContext, backtraceBuffer,
-                                      actualSkippedEntries, *backtraceLength);
-    if (skippedEntries != NULL) {
-        *skippedEntries = actualSkippedEntries;
-    }
-    return backtraceBuffer;
+    if (machineContext != NULL) {
```

Comment:
There's already a check for this a few lines above:

```
444    if (machineContext == NULL) {
445        return NULL;
446     }
```

Reply:
Oh yes good catch. That's a copy-paste error from a rebase. The early-exit check is no longer supposed to be there.